extensions/namespace-lookup/src/test/java/io/druid/server/namespace/cache/JDBCExtractionNamespaceTest.java

```diff
@@ -426,7 +426,7 @@ public void testSkipOld()
     assertUpdated(extractionNamespace.getNamespace(), "foo", "bar");
   }
 
-  @Test(timeout = 60_000L)
```

Comment:
`@Ignore` is better than commenting out.

Also any idea why this test is sketchy? It would be better to fix it than ignore it as it is hard to be confident in the feature without a working test